docs/videos.md

```diff
@@ -0,0 +1,18 @@
+## How to build skipper
+
+We expect you to have [Go](https://golang.org/dl) and [glide](https://github.com/Masterminds/glide) installed.
+
+### How to build skipper without plugins
+
+<script src="https://asciinema.org/a/5eenqvXBDqjQAB2ZcV6mQ7OGj.js" id="asciicast-5eenqvXBDqjQAB2ZcV6mQ7OGj" data-rows="30" async></script>
```

Comment:
let's do it, but on the long term i believe it would nice to the users of our docs if don't load external scripts. This post states that self-hosting is possible: http://blog.asciinema.org/post/self-hosting/